### PR TITLE
feature/7361 create manual link resolver

### PR DESCRIPTION
**Pull request by WiktoriaPrusik**
it is connected with: 
https://github.com/G1ANT-Robot/G1ANT.Language/pull/2
https://github.com/G1ANT-Robot/G1ANT.Addon/pull/108
https://github.com/G1ANT-Robot/G1ANT.Addon.Core/pull/5
https://github.com/G1ANT-Private/G1ANT.Website/pull/334
https://github.com/G1ANT-Robot/G1ANT.Manual/pull/37

### CODE REVIEW

File: For Advanced/050.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-In fact, there are 11 “drawers” — or fields — in each email element: `id`, `from`, `to`, `cc`, `bcc`, `subject`, `content`, `date`, `priority`, `attachments`, `isreply`. Their names are pretty much self-explanatory, but if you need details, just refer to the [mail structure](G1ANT.Language/G1ANT.Language/Structures/MailStructure.md) description.
+In fact, there are 11 “drawers” — or fields — in each email element: `id`, `from`, `to`, `cc`, `bcc`, `subject`, `content`, `date`, `priority`, `attachments`, `isreply`. Their names are pretty much self-explanatory, but if you need details, just refer to the [mail structure](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/MailStructure.md) description.
```
Comment (MarekKosinski) [inline For Advanced/050.md:5]: The link does not work.

File: For Advanced/060.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Providing any credentials in plain text in your code is not reasonable, to say at least. It would be great to mask this data somehow — or better store it in a safe place and retrieve it when necessary without revealing it.
 
-G1ANT.Robot provides such a safe storage for all sensitive information: [Credential Container](g1ant.robot-window/auxiliary-windows/credential-container.md) is a tool for entering and storing data that shouldn’t be visible to everyone, who has an access to a robot script.
+G1ANT.Robot provides such a safe storage for all sensitive information: [Credential Container](https://manual.g1ant.com/link/G1ANT.Tutorials/For%20Advanced/g1ant.robot-window/auxiliary-windows/credential-container.md) is a tool for entering and storing data that shouldn’t be visible to everyone, who has an access to a robot script.
```
Comment (MarekKosinski) [inline For Advanced/060.md]: The link does not work.

File: For Beginners/200.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Try Again
 
-If you take a look at various commands’ description, you will notice that a vast majority of them share the [same arguments](appendices/common-arguments.md). Four of these arguments are targeted at error handling: `errorcall`, `errorjump`, `errormessage` and `errorresult`. You can use one of them to convince a user to provide a correct input.
+If you take a look at various commands’ description, you will notice that a vast majority of them share the [same arguments](https://manual.g1ant.com/link/G1ANT.Tutorials/For%20Beginners/appendices/common-arguments.md). Four of these arguments are targeted at error handling: `errorcall`, `errorjump`, `errormessage` and `errorresult`. You can use one of them to convince a user to provide a correct input.
```
Comment (MarekKosinski) [inline For Beginners/200.md]: Does not work.

**Review by MarekKosinski — APPROVED** [review]
I approve, although 3 of the changed links does not work - Wiktoria said on slack that it is Jacek's fault and will be fixed by him.

File: For Advanced/060.md (hunk 2 above)
Comment (Vegebutcher) [reply For Advanced/060.md]: wrong logic for this link - Manual repo, not Tutorial

File: For Beginners/200.md (hunk 3 above)
Comment (Vegebutcher) [reply For Beginners/200.md]: same as above: wrong logic for this link - Manual repo, not Tutorial

File: For Advanced/060.md (hunk 2 above)
Comment (Vegebutcher) [inline For Advanced/060.md]:
```suggestion
G1ANT.Robot provides such a safe storage for all sensitive information: [Credential Container](https://manual.g1ant.com/link/G1ANT.Manual/g1ant.robot-window/auxiliary-windows/credential-container.md) is a tool for entering and storing data that shouldn’t be visible to everyone, who has an access to a robot script.
```

File: For Beginners/200.md (hunk 3 above)
Comment (Vegebutcher) [inline For Beginners/200.md]:
```suggestion
If you take a look at various commands’ description, you will notice that a vast majority of them share the [same arguments](https://manual.g1ant.com/link/G1ANT.Manual/appendices/common-arguments.md). Four of these arguments are targeted at error handling: `errorcall`, `errorjump`, `errormessage` and `errorresult`. You can use one of them to convince a user to provide a correct input.
```

**Review by Vegebutcher — APPROVED** [review]

File: For Advanced/050.md (hunk 1 above)
Comment (WiktoriaPrusik) [reply For Advanced/050.md:5]: this link will work in the future

**Pull request merged**